vendor/README.md

```diff
@@ -91,8 +91,8 @@ Steps:
 mkdir -p tempLib/org/apache/beam
 
 # Copy files (jar/poms/metadata) to your beam repository
-cp -R ~/.m2/repository/org/apache/beam/beam-vendor-grpc-1_54_0/ \
-      tempLib/org/apache/beam
+cp -R ~/.m2/repository/org/apache/beam/beam-vendor-grpc-1_54_0 \
```

Comment:
Fix the command on macOS: https://www.autodesk.com/support/technical/article/caas/sfdcarticles/sfdcarticles/The-cp-R-command-works-differently-on-Mac-OS-X-than-on-Linux.html#:~:text=The%20cp%20%2DR%20command%20on,directory%20path%20with%20a%20slash.